app/controllers/api/v2/diversity/build_diversity_responses_controller.rb

```diff
@@ -5,14 +5,9 @@ module V2
     module Diversity
       class BuildDiversityResponsesController < ::ApplicationController
         def create
-          p = build_diversity_response_params
+          p = build_diversity_response_params.merge(command: 'CreateDiversityResponse').to_h.symbolize_keys
```

Comment:
This will get refactored in the next bit - all controllers are starting to look the same now - so I might do a helper method to call in the actions or something